src/include/session.h

```diff
@@ -325,12 +325,12 @@ struct __wt_session_impl {
     TAILQ_HEAD(__dhandles_hash, __wt_data_handle_cache) * dhhash;
 
 /* Generations manager */
-#define WT_GEN_CHECKPOINT 0 /* Checkpoint generation */
-#define WT_GEN_COMMIT 1     /* Commit generation */
-#define WT_GEN_EVICT 2      /* Eviction generation */
-#define WT_GEN_HAZARD 3     /* Hazard pointer */
-#define WT_GEN_SPLIT 4      /* Page splits */
-#define WT_GENERATIONS 5    /* Total generation manager entries */
+#define WT_GEN_CHECKPOINT 0   /* Checkpoint generation */
+#define WT_GEN_HAS_SNAPSHOT 1 /* Snapshot generation */
+#define WT_GEN_EVICT 2        /* Eviction generation */
+#define WT_GEN_HAZARD 3       /* Hazard pointer */
+#define WT_GEN_SPLIT 4        /* Page splits */
+#define WT_GENERATIONS 5      /* Total generation manager entries */
```

Comment:
This list is no longer alphabetical - it should be.

Reply:
Fixed in ab154f8f95999594224485f115c27d2b986d5cf4